### ISSUE TITLE
How do i download shot_summe.mat and shot_tvsum.mat?

### ISSUE THREAD

**Issue by linjingxu** [mayu-ot/rethinking-evs#4]
I appreciate your work! But i have a question. How do i download shot_summe.mat and shot_tvsum.mat?

**Comment by sylvainma** [comment]
@linjingxu did you find the link?

**Comment by linjingxu** [comment]
not yet. Are you also interested in the video summarization？

**Comment by linjingxu** [comment]
@sylvainma Not yet. Are you also interested in the video summarization？

**Comment by sylvainma** [comment]
Yes, with my team we are working on a general video summarization repository [here](https://github.com/sylvainma/Summarizer/) implementing most of models and using evaluation recommendations from this paper.

We can share knowledge if you want to.

**Comment by linjingxu** [comment]
@sylvainma Well, I think we can share with each other. My email is linjingxu2019@email.szu.edu.cn

**Comment by mayu-ot** [comment]
I'm sorry for not checking this repo for a long time.
I borrowed shot segmentation files by dppLSTM, but the original files look removed.
I'm not sure if I'm allowed to distribute their files.
I'll try to find a way to solve this issue.

(FYI) We may try KTS  implementations by [MED summaries](http://lear.inrialpes.fr/people/potapov/med_summaries) or [this repo.](https://github.com/TatsuyaShirakawa/KTS) on GoogLeNet features to reproduce this.

**Comment by MHGCC** [comment]
Hello.
Is this problem solved?
Or may I ask who can provide the new data download address ？
Or I want to know what are the elements of shot_tvsum.mat/shot_summe.mat.

**Comment by mayu-ot** [comment]
Hello,

This repository provides data including temporal segmentation that looks to be generated using KTS on GoogLeNet features.
You can also use the data here.
https://github.com/e-apostolidis/PoR-Summarization-Measure/tree/master/data
"Performance over Random: A Robust Evaluation Protocol for Video Summarization Methods" (MM '20)

By the way, this paper discusses the impact of data splits on the video summary evaluation.
If you are trying to evaluate your own methods with TVSum or SumMe, this paper is also good to check.

**Comment by MHGCC** [comment]
thank you for your reply.
